### DISCUSSION TITLE
How to make the layout left across screen?

### DISCUSSION THREAD

**Discussion by OriLiMu** [qtile/qtile#4481] [Q&A]
for example I have two screen A(with window WinA on it), B(with WinB on it). 
Screen A on the left of screen B, now I focus on WinB, I want to focus on WinA by press <kbd>Mod + left</kbd> . 
Related function is `lazy.layout.left()`(just like i3wm) but seems this function cant across screen. 
I check the related doc here: https://docs.qtile.org/en/stable/manual/commands/api/layouts.html. 
But not find related info. 
How to fix this issue.

**Comment by elParaguayo** [comment]
You probably want this code: https://github.com/qtile/qtile-examples/blob/master/traverse.py
